fakeserver/routes.go

```diff
@@ -118,6 +145,29 @@ func getV1SplitRegistry() (interface{}, error) {
 	return splitRegistry, nil
 }
 
+func getV2SplitRegistry() (interface{}, error) {
```

Comment:
Can this return `(v2SplitRegistry, error)`?

Reply:
oh because this thing is literally being consumed by the web server got it.

Reply:
so like we could decompose into a typed func that the get method uses and stuff, but I don't care if you don't :) I may have modeled this pattern already, I forget.

Reply:
yeah, because the router expects a return value of `interface{}`, all these functions must use that same signature.

a bit of googling suggested there may be a way to use reflection to allow these functions to have a concrete return value.  but tbh, i can't quite wrap my head around how to get that right.